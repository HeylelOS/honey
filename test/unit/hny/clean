#!/bin/sh

set -e

"${TEST_BIN}" clean 'arxiv'

