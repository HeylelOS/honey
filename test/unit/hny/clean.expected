clean: Starting for 'arxiv'
Test Archive - Clean
